moveit_planners/ompl/ompl_interface/src/constraints_library.cpp

```diff
@@ -605,4 +605,5 @@ ompl::base::StateStoragePtr ompl_interface::ConstraintsLibrary::constructConstra
 
     return sstor;
   }
+#warning "else case not yet handled: missing return value!"
```

Comment:
@davetcoleman This function is missing a return value. However, I'm not sure what to return here / how to handle the error(?) case. As you are mentioned as a coauthor, you probably know better.